### PR TITLE
Bump maya version --> 0.7.0

### PR DESCRIPTION
**Pull request by prateekpandey14**
1. Change the maya version to 0.7.0
2. This commit added to get the version details if maya-apiserver is not running in environment or if we running mayactl commands outside of the maya-apiserver POD to get the actual error if there.
 
```sh
mayactl version
Unable to connect to mapi server address
Version: v0.7.0-unreleased
Git commit: 90492fd8503a35fbfd21789829d39820129bdeb3
GO Version: go1.10.3
GO ARCH: amd64
GO OS: linux
m-apiserver url:  http://127.0.1.1:5656
m-apiserver status:  not reachable
Provider:  Unknown
```
```sh
Version: v0.7.0-unreleased
Git commit: 90492fd8503a35fbfd21789829d39820129bdeb3
GO Version: go1.10.3
GO ARCH: amd64
GO OS: linux
m-apiserver url:  http://127.0.1.1:5656
m-apiserver status:  Running
Provider:  Unknown
```
```sh
prateek:maya$ mayactl volume list
Unable to connect to mapi server address
Get http://127.0.1.1:5656/latest/volumes/: dial tcp 127.0.1.1:5656: connect: connection refused
```
Signed-off-by: prateekpandey14 <prateekpandey14@gmail.com>

### CODE REVIEW

File: cmd/mayactl/app/command/commands.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func NewMayaCommand() *cobra.Command {
 				mapiserver.Initialize()
 				if mapiserver.GetConnectionStatus() != "running" {
 					fmt.Println("Unable to connect to mapi server address")
-					os.Exit(1)
+					//os.Exit(1)
```
Comment (kmova) [inline cmd/mayactl/app/command/commands.go]: why this change?

Comment (prateekpandey14) [reply cmd/mayactl/app/command/commands.go]: Updated the PR description and added the code comment too.
